lib/ClusterShell/RangeSet.py

```diff
@@ -194,6 +198,14 @@ def fromone(cls, index, pad=0, autostep=None):
             inst.add_range(index.start or 0, index.stop, index.step or 1, pad)
         return inst
 
+    @property
+    def padding(self):
+        warnings.warn("padding has been deprecated", DeprecationWarning)
```

Comment:
i prefer we don't emit warning right now. if you update CS and some user app, like Shine, uses this padding args, there will be warnings everywhere without giving a chance to these apps to be compatible.

Reply:
Actually, forget about the PendingDeprecationWarning thing. We don't need it yet. I think the padding property might still be useful in some cases, even with mixed length padding... so perhaps this can just stay as part of the API (especially useful to get the max padding length of a set I guess). If needed, we can optimize its performance later.